### PR TITLE
`contracts_optimism_contract_creator_list` Additional Edits

### PR DESCRIPTION
**Pull request by kaiblade**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

File: models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql [hunk 1]
```diff
@@ -617,16 +617,45 @@ WITH curated_list AS (
...
-  FROM curated_list f
-WHERE f.creator_address NOT IN (
-   SELECT creator_address FROM {{ ref('contracts_optimism_deterministic_contract_creators') }}
+,filtered_list AS (
+  SELECT creator_address, contract_project
+  --filter out creators that we never want to map
+    FROM (
+      SELECT creator_address, contract_project FROM curated_list cl
+      UNION ALL
+      SELECT creator_address, contract_project FROM mapped_list ml
```
Comment (jeff-dude) [inline models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql:644]: @kaiblade what is this `mapped_list`? there is no CTE in the model named that. i imagine this is the issue

Comment (kaiblade) [reply models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql]: Yes, I think that is the issue as well, I wonder how that got in. I will correct it and make another commit, hopefully, the issue will be fixed. I will let you know if the issue has been fixed or not. Thank you very much

Comment (kaiblade) [reply models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql]: It has been fixed. Thank you for the assistance.

Comment (jeff-dude) [reply models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql]: nice!

**Comment by jeff-dude** [comment]
tagging @MSilb7 for a thumbs up 👍

File: models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql [hunk 2]
```diff
@@ -617,16 +617,43 @@ WITH curated_list AS (
       ,(0x9aaa603cb8ce688cdb48ab132dd1b8ad7b5ade02, 'Bulls DAO')
       ,(0x9bbFB9919062C29a5eE15aCD93c9D7c3b14d31aa, 'AttestationStation')
       ,(0xC8D7d0BD61C5Ca5A493a229f6754Da5560F486ae, 'Magpie')
-      
+      ,(0x344a908d1a7b7d06b7ad7169c1db81fc9d496de9, 'zkLink')
+      ,(0xd0fe1538031f5647b1846d35b36f22e7317a1e35, 'Altitude Bridge')
+      ,(0x645487828d5769c20365fdb8786a5e2b734d785c, 'Altitude Bridge')
+      ,(0x01cf911dd46f90290bf5f93d0ef34237df7df1b6, 'Hedgey Finance')
+      ,(0x320bcb681ce7023edfe48ade9cf5bf67a11bcd36, 'Hedgey Finance')
+      ,(0xf537880c505bfa7cda6c8c49d7efa53d45b52d40, 'Binance Exchange')
```
[the remark this answers is not in the document]
Comment (kaiblade) [reply models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql:625]: Okay, will do that.

File: models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql [hunk 3]
```diff
@@ -617,16 +617,43 @@ WITH curated_list AS (
...
 WHERE f.creator_address NOT IN (
    SELECT creator_address FROM {{ ref('contracts_optimism_deterministic_contract_creators') }}
 )
 
 GROUP BY 1,2
+)
 
+-- Enforce consistent project name mapping to contracts_optimism_project_name_mappings.sql
+SELECT list.creator_address,
+  (CASE
```
[the remark this answers is not in the document]
Comment (kaiblade) [reply models/contracts/optimism/contracts_optimism_contract_creator_address_list.sql:652]: Ok, will do as requested.

**Comment by kaiblade** [comment]
@MSilb7 I've made all the requested changes. This PR solves the "Contract deployer mapping" task on Dework here: https://app.dework.xyz/optimism-community/main-space-35638?taskId=9906bf4d-f0ef-45bc-8bb5-f3e0444ec2a1. If this PR is approved and merged, Is there any info to add (e.g link to PR) when submitting the task on Dework as no such information is provided in the link?

**Review by jeff-dude — APPROVED** [review]
thank you for contributing!

**Pull request merged**